packages/server/graphql/mutations/setTaskEstimate.ts

```diff
@@ -90,6 +92,16 @@ const setTaskEstimate = {
     let githubLabelName: string | undefined = undefined
     const {integration} = task
     const service = integration?.service
+
+    if (!stageId || !meeting) {
```

Comment:
-1 I think this mutation needs some cleanup. At the moment if no `meetingId` is provided, always an error is returned. So `meetingId` should be mandatory in the GraphQL schema.

Before GitHub also allowed to push estimates in some combinations without meeting, but I don't think we need to support this for GitHub alone, but rather should add it for all integrations if the need arises.

Reply:
Made `meetingId` non-nullable